src/openforms/formio/components/custom.py

```diff
@@ -106,7 +106,11 @@ def normalizer(component: Component, value: str) -> str:
 
 class FamilyMembersHandler(Protocol):
     def __call__(
-        self, bsn: str, include_children: bool, include_partner: bool
+        self,
+        bsn: str,
+        include_children: bool,
+        include_partners: bool,
```

Comment:
please only make the minimal fix, I see the actual arg being used in the call on line 184 is different from this signature, but refrain from refactoring.